sormas-app/app/src/main/res/layout/fragment_event_edit_layout.xml

```diff
@@ -105,6 +106,12 @@
                 app:enumClass="@{eventManagementStatusClass}"
                 app:value="@={data.eventManagementStatus}"/>
 
+            <de.symeda.sormas.app.component.controls.ControlSwitchField
```

Comment:
I would recommend to use a ControlSpinnerField instead because the captions of the identification source values could get quite large, which could lead to layout problems